WordPress/src/main/java/org/wordpress/android/util/WPVideoUtils.java

```diff
@@ -106,6 +115,51 @@ public static MediaComposer getVideoOptimizationComposer(@NonNull Context ctx, @
         return mediaComposer;
     }
 
+    // TODO: this should replace the equivalent function used for m4m lib once we fully introduce the Mp4Composer lib
+    public static ComposerInterface getVideoOptimizationComposer(@NonNull String inputFile,
+                                                                 @NonNull String outFile,
+                                                                 @NonNull Listener listener,
+                                                                 int width, int bitrate) {
+        ComposerInterface composer = ComposerProvider.INSTANCE.getComposerForUseCase(new CompressVideo(
+                inputFile,
+                outFile,
+                VideoFormatMimeType.AVC,
+                bitrate * 1024,
+                2,
+                96 * 1024,
```

Comment:
Could we maybe move these to constants and add a comment next to each one, explaining the reasoning why these values are passed like this? 
Also wondering, could it be possible that we need to pass any other values or will this work universally?

Reply:
Not a great study behind that tbh 😄 ....I basically just tried to use the set of parameters that were used (as a fixed set like here) in the previous implementation. Added a comment in the code that hope can help to clarify, let me know 🙇

Reply:
thanks for your answer! sounds good 👍